src/SamplesApp/UITests.Shared/Windows_Devices/Midi/MidiDeviceOutput.xaml.cs

```diff
@@ -0,0 +1,758 @@
+			// Set default to disable Send button for any message types that fall through
+			switch (_currentMessageType)
+			{
+				case MidiMessageType.NoteOff:
+				case MidiMessageType.NoteOn:
+				case MidiMessageType.PolyphonicKeyPressure:
+				case MidiMessageType.ControlChange:
+
+					if (parameter3SelectedIndex != -1)
+					{
+						sendButton.IsEnabled = true;
+					}
+					break;
+
+				default:
+					sendButton.IsEnabled = false;
+					break;
+			}
+
+			switch (_currentMessageType)
+			{
+				case MidiMessageType.NoteOff:
+				case MidiMessageType.NoteOn:
+					// This list is for Velocity Values, of which there are 128
+					PopulateParameterList(parameter3, 128, "Velocity");
+					break;
+
+				case MidiMessageType.PolyphonicKeyPressure:
+					// This list is for Pressure Values, of which there are 128
+					PopulateParameterList(parameter3, 128, "Pressure");
+					break;
+
+				case MidiMessageType.ControlChange:
+					// This list is for Values, of which there are 128
+					PopulateParameterList(parameter3, 128, "Value");
+					break;
+
+				default:
+					// Start with a clean slate
+					// Hide the first parameter
+					parameter3.Header = "";
+					parameter3.IsEnabled = false;
+					parameter3.Visibility = Visibility.Collapsed;
+					break;
+			}
+		}
+
+#if !WINDOWS_UWP
+		/// <summary>
+		/// Helper function to populate a dropdown lists with options
+		/// </summary>
+		/// <param name="list">The parameter list to populate</param>
+		/// <param name="numberOfOptions">Number of options in the list</param>
+		/// <param name="listName">The header to display to the user</param>
+		private void PopulateParameterList(NumberBox numberBox, int numberOfOptions, string listName)
+		{
+			numberBox.Maximum = numberOfOptions;
+
+			// Show the list, so that the user can make the next choice
+			numberBox.Header = listName;
+			numberBox.IsEnabled = true;
+			numberBox.Visibility = Visibility.Visible;
+		}
+
+		private int GetParameterValue(NumberBox numberBox) => (int)numberBox.Value;
+#else
+		private void PopulateParameterList(TextBox numberBox, int numberOfOptions, string listName)
+		{
+			// Show the list, so that the user can make the next choice
+			numberBox.Header = listName;
+			numberBox.IsEnabled = true;
+			numberBox.Visibility = Visibility.Visible;
+		}
+
+		private int GetParameterValue(TextBox numberBox) => int.Parse(numberBox.Text);
+#endif
+
+		private void NotifyUser(string message)
+		{
+			statusBlock.Text = message;
+		}
+
+
+		#region Jingle Bells
+		private byte ENote = 64;
+		private byte DNote = 62;
+		private byte CNote = 60;
+		private byte FNote = 65;
+		private byte GNote = 67;
+		private const int Skip = 400;
+
+		private async void JingleBells_Click(object sender, RoutedEventArgs args)
+		{
+			if (_currentMidiOutputDevice != null)
+			{
+				var midiOutputQueryString = MidiOutPort.GetDeviceSelector();
+				var midiOutputDevices = await DeviceInformation.FindAllAsync(midiOutputQueryString);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'midiOutputDevices'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)